### PR TITLE
Required field indicators

### LINKED ISSUE #169: Contact UX: Make it more obvious a field is required
It's odd that you can mark certain fields as required when there's no visual difference on the front end of a required vs a not required field. How is the user meant to know that a field is required prior to submitting the form? It might be a good idea to add an asterisk (it's basically a standard at this point) to required labels to make it a little more obvious of this.

### PR DESCRIPTION
**Pull request by braamgenis**
Resolves #169 
Option to display asterisks on required fields and displays legend above the form when it's enabled.
E.g.
required fields

### CODE REVIEW

File: widgets/contact/contact.php [hunk 1]
```diff
@@ -785,6 +790,14 @@ function render_form_fields( $fields, $errors = array(), $instance ){
 		$field_ids = array();
 		$label_position = $instance['design']['labels']['position'];
 
+		$indicate_required_fields = $instance['settings']['required_field_indicator'];
+
+		if( ! empty( $indicate_required_fields ) ) {
+		    ?>
+            <p><em><?php esc_html_e( 'Fields marked with * are required', 'so-widgets-bundle' ) ?></em></p>
```
Comment (gregpriday) [inline widgets/contact/contact.php]: I'd prefer to have this user configurable. We can make this text the default, but this is something that a lot of users are going to want to change (for translation and personal preference).

File: widgets/contact/contact.php [hunk 2]
```diff
@@ -799,11 +812,15 @@ function render_form_fields( $fields, $errors = array(), $instance ){
 
 			?><div class="sow-form-field sow-form-field-<?php echo sanitize_html_class( $field['type'] ) ?>"><?php
 
+            $label = $field['label'];
```
Comment (gregpriday) [inline widgets/contact/contact.php]: Line 815 and 817 are being indented with spaces, not tabs.

File: widgets/contact/contact.php (hunk 1 above)
Comment (braamgenis) [inline widgets/contact/contact.php]: I thought about doing that... can't remember why I didn't. ¯\_(ツ)_/¯

File: widgets/contact/contact.php (hunk 2 above)
Comment (braamgenis) [inline widgets/contact/contact.php]: :sob:

**Comment by braamgenis** [comment]
@gregpriday Cool, fixed those up. Ready to merge when you can. :+1:

**Pull request merged**